test/frontend/util/datetime.test.js

```diff
@@ -11,11 +11,11 @@ describe('datetime', () => {
 
   describe('duration()', () => {
     it('shows a human readable duration when provided two dates', () => {
-      expect(duration(startTime, endTime)).to.equal('4 minutes');
+      expect(duration(startTime, endTime)).to.equal('3 minutes');
     });
 
     it('works if endTime isn\'t defined', () => {
-      expect(duration(+new Date())).to.equal('less than a minute');
+      expect(duration(+new Date())).to.equal('0 seconds');
```

Comment:
The "strict" formatter is a little shorter, but is more precise. date-fns doesn't offer the floor rounding method on the regular formatter, and before, the two duration formatter time utils would give different results for the same duration.